code-challenges.test.js

```diff
@@ -54,3 +120,26 @@ var numbersToAdd3 = []
 
 
 // b) Create the function that makes the test pass.
+// going to reuse almost all of my fibonacci sequence code
+// numbers are the same, except do not provide a starting array
+// i lied, its not the same
+// need to store first index of array as newArr[0] = array[0]
+// need to add first index of newArr to next number in array
+
+
+const addNum = array.map(value => newArr[i] + array[i +1])
+
+const addNum = (array) => {
+    let newArr = []
+    if (array !== []){
+        newArr = [array[0]]
+        for (let i = 0; i < array.length - 1; i++) {
+            newArr.push(newArr[i] + array[i + 1])
+        }
+    } else{
+        return array
+    }
+    return newArr
+}
```

Comment:
Nice job. As a refactor, I don't think you need an if/else statement with the logic you created. If given an empty array, your logic will return an empty array.